docs/administration/vulnerability-scanning/scan-individual-artifact.md

```diff
@@ -69,3 +69,30 @@ When scanning an OCI image index, Harbor will send scan requests for each of the
 
 
 When an OCI image index is successfully scanned, the summary of the vulnerability report for the OCI image index is aggregated from the individual scan results of the the artifacts referenced by the index. The vulnerability report will show both sets of statistics.
+
+## Export Vulnerability Scans
+
+As a project administrator, you are able to export vulnerability scan results of one or more of your projects as CVS and download it from your Harbor instance.
```

Comment:
As a `project admin or maintainer or developer`
only able to export one project at one time

Reply:
updated